### DISCUSSION TITLE
option to disable afk fishing

### DISCUSSION THREAD

**Discussion by Tizooo** [PurpurMC/Purpur#630] [__Dismissed__] [answered]
![Unbenannt](https://user-images.githubusercontent.com/34001316/131329320-d979f8c9-49c2-4e21-904d-46df5a8a3805.PNG)
maybe fix this afk fishing farm? or a option to disable it

**Comment by BillyGalbreath** [comment, marked as answer]
This is better suited for an anti-cheat plugin. Here's one I found with a quick google search https://www.spigotmc.org/resources/anti-auto-fishing.58978/
